dll/win32/msgina/gui.c

```diff
@@ -56,28 +56,34 @@ DlgData_Create(HWND hwndDlg, PGINA_CONTEXT pgContext)
     return pDlgData;
 }
 
-static BOOL
-DlgData_LoadBitmaps(PDLG_DATA pDlgData)
+static void
```

Comment:
How would declaring the function as VOID be considered as an improvement? Could you elaborate?

Reply:
Case fixed.
The current return value is not what is wanted, then it becomes unused.